src/main/java/javaslang/collection/List.java

```diff
@@ -133,13 +136,19 @@
             @SuppressWarnings("unchecked")
             final List<T> list = (List<T>) elements;
             return list;
-        } else if (elements instanceof ArrayList || elements instanceof Vector) {
-            @SuppressWarnings("unchecked")
-            final java.util.List<T> indexedList = (java.util.List<T>) elements;
+        } else if (elements instanceof java.util.List<?>) {
```

Comment:
Hi Sebastian,
just a question - is there an advantage providing a `<?>` when using `instanceof`? I haven't seen it before. Also javac does not complain when omitting it (using `-Xlint:all -Werror`).